synapseclient/table.py

```diff
@@ -866,9 +869,142 @@ def _filter_duplicate_columns(self, syn, columns_to_add):
         return valid_columns
 
 
+class SubmissionViewSchema(SchemaBase):
```

Comment:
Unsure how i should test this?

Reply:
If we can consolidate it per your suggestion above then it lessens the need for separate tests.

Adding a usage of a submission view to an integration test in test_evaluations.py would be good though.